### DISCUSSION TITLE
Remote thermostat with FW120

### DISCUSSION THREAD

**Discussion by xavp06** [emsesp/EMS-ESP32#1984] [General]
Hi, 
I get yesterday my EMS gateway (S3 from BBQKees), and connected it to the EMS bus. I have a FW120 thermostat controlling an Odealis Condens boiler from ELM_Leblanc (Heatronic 3), used for hot water and a floor heating system. The problem is that the thermostat is in the boiler room, and therfore the measured temperature has nothing to do with the desired one. Previous owner has connected a Netatmo thermostat to the boiler, which is only controling it ON/OFF. not even sure how the connected FW120 is acting in this scenario. 
I want to remove that proprietary Netatmo system, and include the boiler into my domotic system (Jeedom or Home Assistant, not fully defined yet). 
So in short I was very interested in the remote thermostat feature, and the documentation is explaining that with a FW120 a remote FB10 will be created. 
I tried to follow docuemntation, and some discussions hear to make it work with the stable 3.6.5. I could not get it work. the command thermostat/hc1/remotetemp does not exist. it is  a readonly info but not a command. 
Then I installed the latest build (3.7.0-dev34), and oh ! it works ! FB10 is created and I can change the remote temperature. 
It might be good to update the documentation to indicate in which version this FW120/FB10 is really implemented and functional. The list of commands and info seems to have changed quite a bit between 3.6.5 and 3.7.0. 

Is there anyone here with some experience on such a configuration? There are few controls I'm not completely sure how to setup to have the system working by considering the external temperature probe, and the remote room temperature information I will regularly push to the EMS with MQTT.

**Comment by proddy** [comment]
Yes, lots of changes between 3.6.5 and 3.7.0 and the latest dev version is soon to be released, and then we'll update the documentation
